insights/parsers/sys_bus.py

```diff
@@ -0,0 +1,70 @@
+"""
+``/sys/bus`` Device Usage Information
+=====================================
+
+A parser to parse the usage information of devices connected
+on sys/bus.
+
+Parsers included in this module are:
+
+CdcWDM - file ``/sys/bus/usb/drivers/cdc_wdm/module/refcnt``
+------------------------------------------------------------
+
+"""
+
+
+from insights import parser, Parser
+from insights.parsers import SkipException, ParseException
+from insights.specs import Specs
+
+
+@parser(Specs.cdc_wdm)
+class CdcWDM(Parser):
+
+    """
+    This file `/sys/bus/usb/drivers/cdc_wdm/module/refcnt` contains
+    device usage count, i.e if a device is in use then the non-zero
+    value will be present in the file.
+
+    Sample Content:
```

Comment:
Check the rendered docs, I believe you need a `::` here to keep the following 2 lines separate.

Reply:
Done